apps/src/weblab/WebLab.js

```diff
@@ -258,29 +314,38 @@ WebLab.prototype.onIsRunningChange = function () {
 };
 
 /**
- * Load the asset list and store it as this.currentAssets
+ * Load the file entry list and store it as this.fileEntries
  */
-WebLab.prototype.loadCurrentAssets = function () {
-  assetsApi.ajax('GET', '', xhr => {
+WebLab.prototype.loadFileEntries = function () {
+  filesApi.ajax('GET', '', xhr => {
     let parsedResponse;
     try {
       parsedResponse = JSON.parse(xhr.responseText);
     } catch (e) {
-      console.error('assets API parse failed, error: ' + e);
-      this.currentAssets = null;
+      console.error('files API parse failed, error: ' + e);
+      this.fileEntries = null;
       return;
     }
-    assetListStore.reset(parsedResponse);
-    this.currentAssets = assetListStore.list().map(asset => ({
-      name: asset.filename,
-      url: '/v3/assets/' + dashboard.project.getCurrentId() + '/' + asset.filename
+    assetListStore.reset(parsedResponse.files);
+    this.fileEntries = assetListStore.list().map(fileEntry => ({
+      name: fileEntry.filename,
+      url: `/v3/files/${dashboard.project.getCurrentId()}/${fileEntry.filename}`
```

Comment:
Should the exact structure of this URL live with the `filesApi` and be generated through a helper?